packages/react/src/pages/index.tsx

```diff
@@ -0,0 +1,6 @@
+export { default as CRMContactList } from './crm-contact-list/crm-contact-list';
+
+// next pages
```

Comment:
Are commented blocks of code necessary here?

Reply:
I removed comments and added empty future pages